manifest/manifest.go

```diff
@@ -186,7 +187,7 @@ func AddDummyV2S1Signature(manifest []byte) ([]byte, error) {
 
 // MIMETypeIsMultiImage returns true if mimeType is a list of images
 func MIMETypeIsMultiImage(mimeType string) bool {
-	return mimeType == DockerV2ListMediaType
+	return mimeType == DockerV2ListMediaType || mimeType == imgspecv1.MediaTypeImageIndex
```

Comment:
Please carry over the added tests for OCI from the other PR. https://github.com/containers/image/pull/400#discussion_r163017767